src/main/java/org/coffeemine/app/spring/UserProfileComponent.java

```diff
@@ -0,0 +1,86 @@
+package org.coffeemine.app.spring;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.tabs.Tab;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.component.formlayout.FormLayout;
+
+
+class profileView extends VerticalLayout {
+    public profileView() {
+        this.setWidthFull();
+        this.setMaxWidth("1200px");
+
+        Button back = new Button("back");
+        back.addThemeVariants(ButtonVariant.MATERIAL_CONTAINED);
+        this.add(back);
+
+        // Header
+        H2 title = new H2("Account Name");
+        Icon user = VaadinIcon.USER.create();
+        user.setSize("5em");
+        add(back);
+        add(user);
+        this.add(title);
+
+        HorizontalLayout info = new HorizontalLayout();
+        info.setWidthFull();
+        this.add(info);
+
+        VerticalLayout details = new VerticalLayout();
+        details.setWidthFull();
+        details.add(new H3("Account details"));
+
+        FormLayout accountDetails = new FormLayout();
+        accountDetails.addFormItem(new TextField(), "Full name:");
+        accountDetails.addFormItem(new TextField(), "Email:");
+        details.add(accountDetails);
+
+        VerticalLayout states = new VerticalLayout();
+        states.setWidthFull();
+        states.add(new H3("Account states"));
+
+        FormLayout accountStates = new FormLayout();
+        TextField salaryPerHourFiled = new TextField();
+        salaryPerHourFiled.setPlaceholder("Salary per hour");
+        accountStates.addFormItem(salaryPerHourFiled, "Salary per hour:");
```

Comment:
These arent supposed to be TextFields, use Span instead.

Reply:
fix it already